### DISCUSSION TITLE
Welcome to bt-studio Discussions!

### DISCUSSION THREAD

**Discussion by jmplaza** [JdeRobot/bt-studio#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nazerek** [comment]
Hello! My name is Nazerke.   I am a beginner open-source developer. My background is in Robotics and I am interested in educational, academic projects. I plan to take part in them, exchange experience and meet like-minded people 👋

**Comment by jmplaza** [comment]
HI @Nazerek ,

the best way to start contributing is using BT-Studio :-) , betatesting it and even sending PullRequests with solutions to the issues.

Cheers,
